src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/TransformFunctions.kt

```diff
@@ -14,3 +28,155 @@ fun transform(testData: EntityTestData): ModelTestData {
     value = testData.value
   )
 }
+
+fun transformToCreateResponse(licence: EntityLicence): CreateLicenceResponse {
+  return CreateLicenceResponse(licence.id, licence.typeCode, licence.statusCode)
+}
+
+fun transform(createRequest: CreateLicenceRequest): EntityLicence {
```

Comment:
We can consider using the ModelMapper library for this, which maps one object to another easily as long as field names are the same, and it can accept custom mappings if the field names are different

Reply:
That's a good idea - can we try this on one object in the next PR?

Reply:
👍🏼